data/registry/instrumentation-go-honeycomb.yml

```
@@ -0,0 +1,11 @@
+title: Honeycomb.io OpenTelemetry Distro for Go
+registryType: instrumentation
+isThirdParty: true
+language: go
+tags:
+  - go
+  - instrumentation
+repo: https://github.com/honeycombio/honeycomb-opentelemetry-go
+license: Apache 2.0
+description: Honeycomb's distribution of OpenTelemetry for Go
+authors: Hound Technology Inc
```

Should this be the following (which is what is in the NOTICE files of those repos)?

```suggestion
authors: Honeycomb, Hound Technology, Inc
```

---

Ah, we can include both legal and DBA names 👍